### DISCUSSION TITLE
Documentation for the Supabase Flutter package

### DISCUSSION THREAD

**Discussion by divinemaredi** [supabase/supabase#693] [Questions] [answered]
I saw that a dart client was recently released on [pub.dev](https://pub.dev/packages/supabase), we appreciate the hard work!

But just wondering when can one expect full documentation for this flutter package? ( If an estimation can't be provided currently, that's also fine, we'll wait).

In the meantime, should one use the dart client in comparison with the js documentation?

**Comment by kiwicopple** [comment, marked as answer]
Thanks for the interest on this one @divinemaredi 

We're working with the community on these libraries - there will be several ready for documentation soon (Kotlin, C#, Python, Dart)

**Reply by divinemaredi** [reply]
Great stuff, thanks a lot!
